tests/yam/migration/migration_auto.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
```

Comment:
we have dropped the term "auto" long time ago, let's use 'unattended':
`migration_unattended.pm`